packages/cli/src/common/script.ts

```diff
@@ -126,7 +126,8 @@ export class Script<TContext> {
           throw err
         }, constVoid)
       )
-    } catch (err) {
+    } catch (error) {
+      const err = error as Error
```

Comment:
If this is now required, I wonder if they enabled type annotations in the catch error type:

```suggestion
    } catch (error: Error) {
```

Reply:
yeahh!! that what I expected but not! In fact, it is even weirder you can use `error:any` but you can't use any other type. This is what the official documentation says:

https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#use-unknown-catch-variables